tests/integration/events.bats

```diff
@@ -107,3 +131,145 @@ function teardown() {
   run eval "grep -q 'test_busybox' events.log"
   [ "$status" -eq 0 ]
 }
+
+@test "events --interval 100ms <multiple containers>" {
+  # XXX: currently cgroups require root containers.
+  requires root
+
+  # run two containers
+  runc run -d --console-socket $CONSOLE_SOCKET bb_one
+  [ "$status" -eq 0 ]
+
+  runc run -d --console-socket $CONSOLE_SOCKET bb_two
+  [ "$status" -eq 0 ]
+
+
+  # spawn two sub processes (shells)
+  # the first sub process is an event logger that sends stats events to events.log
+  # the second sub process waits for an event that incudes test_busybox then
+  # kills the test_busybox container which causes the event logger to exit
+  (__runc events --interval 100ms bb_one bb_two >> events.log) &
+  (
+    retry 3 1 eval "grep -q 'bb_two' events.log"
+    teardown_running_container bb_one
+    teardown_running_container bb_two
+  ) &
+  wait # wait for the above sub shells to finish
+
+  [ -e events.log ]
+
+  run cat events.log
+  printf '%s\n' "${lines[@]}"
+  [ "$status" -eq 0 ]
+
+  [ ${#lines[@]} -gt 2 ]
+
+  expect_stats_for_container "bb_one" "${lines[@]}"
+  expect_stats_for_container "bb_two" "${lines[@]}"
+
+  teardown_running_container bb_one
+  teardown_running_container bb_two
+}
+
+@test "events --stats non-existent-container-1 non-existent-container-2 test_busybox (bails when any container doesn't exist)" {
+  # XXX: currently cgroups require root containers.
+  requires root
+
+  runc run -d --console-socket $CONSOLE_SOCKET test_busybox
+  [ "$status" -eq 0 ]
+
+  runc events --stats non-existent-container-1 non-existent-container-2 test_busybox
+  [ "$status" -ne 0 ]
+  [[ "${lines[0]}" =~ "non-existent-container-1" ]]
+  [[ "${lines[0]}" =~ "does not exist" ]]
+  [[ "${lines[1]}" =~ "non-existent-container-2" ]]
+  [[ "${lines[1]}" =~ "does not exist" ]]
+  [[ "${lines[@]}" != *"test_busybox"* ]]
+}
+
+@test "events --interval non-existent-container-1 non-existent-container-2 test_busybox (bails when any container doesn't exist)" {
```

Comment:
This is a duplicate of the previous test.

Reply:
The previous test is making an assertion about the output of `runc events --stats` whereas this one concerns `runc events --interval`